### PR TITLE
Added a new column to display the boostTeam function.

### PR DESCRIPTION
**Pull request by MiguelTrujillo26**
Added a new column to display the boostTeam function in TeamList view.

### CODE REVIEW

File: web/src/views/TeamList.js [hunk 1]
```diff
@@ -143,6 +167,7 @@ const TeamList = () => {
                                     <th rowSpan={2}>Nivel de premiación</th>
                                     <th colSpan={2}>Última potenciación</th>
                                     <th rowSpan={2}>Actualizar Nivel</th>
+                                    <th rowSpan={2}>Impulsar Equipo</th>
```
Comment (fernando-arevalo-galilei) [inline web/src/views/TeamList.js:170]: I wonder if we should rename the column to "Potenciar" or change column "Última potenciación" to "Último impulso". Or even remove column "Última potenciación" and their children. What do you thing @andymotagl

Comment (andymotagl) [reply web/src/views/TeamList.js:170]: It is ok for me having a bit of spanglish here: Último Boost

File: app/src/db/actor.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ const pool = require('./pool');
 const tx = require('pg-tx').default;
 const { BadRequest } = require('../web/apiResponse');
 
-const CREATE_ACTOR_STATEMENT = 'INSERT INTO actor (full_name, email, createdBy, company_id, auth_user_id, rol) VALUES ($1, $2, $3, $4, $5, $6) RETURNING *';
+const CREATE_ACTOR_STATEMENT = 'INSERT INTO actor (full_name, email, created_by, company_id, auth_user_id, rol) VALUES ($1, $2, $3, $4, $5, $6) RETURNING *';
```
Comment (andymotagl) [inline app/src/db/actor.js:5]: Merge main.

**Review by andymotagl — APPROVED** [review]

File: web/src/views/TeamList.js [hunk 3]
```diff
@@ -155,13 +180,18 @@ const TeamList = () => {
                                     <tr key={team.name}>
                                         <td>{team.name}</td>
                                         <td>{levelNames[team.prize_level]}</td>
-                                        <td>{team.boost_started}</td>
-                                        <td>{team.boost_ended}</td>
+                                        <td>{formatDate(team.boost_started, 'dd/MM/yyyy HH:mm')}</td>
+                                        <td>{formatDate(team.boost_ended, 'dd/MM/yyyy HH:mm')}</td>
```
Comment (fernando-arevalo-galilei) [inline web/src/views/TeamList.js:184]: i'm not sure if adding a new dependency is required to format dates. I think you can use something like `team.boost_started.toLocaleString()` or function `formatDate` from `utils/dates/`

Comment (henrymcarvajal-galilei) [reply web/src/views/TeamList.js:184]: there is a format date function in web/src/utils

**Review by henrymcarvajal-galilei — APPROVED** [review]

File: web/src/views/TeamList.js (hunk 1 above)
Comment (MiguelTrujillo26) [reply web/src/views/TeamList.js:170]: To be aligned, the column name could be “Activar Boost” and the row name “Impulsar”.

File: app/src/db/actor.js (hunk 2 above)
Comment (MiguelTrujillo26) [reply app/src/db/actor.js:5]: It was already merged with the main one so I have to delete this change

**Pull request merged**